templates/content/field--oe-document-reference--oe-document--oe-document.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Field template of the oe_document_reference for the oe_document document.
```

Comment:
Still awkward, but I don't have a good counter proposal.
Let's keep it, unless @drishu has a better idea during second review.
(see also the other field templates)

Reply:
My proposal is less scientific but more humane and understandable
"Field template of the Document reference entity."
We use the label not machine name this way we can understand.

Reply:
Actually this was Carlos first version, just in lowercase.
My initial request here was to include the field name, bundle and entity type, so we describe what we target.
But it is hard to put that into a single line.

One option would be to put these explanations in a separate block below:
```suggestion
 * Template for the 'Document' field in the 'Document reference' entity type.
 *
 * Entity type: 'oe_document_reference'.
 * Field: 'oe_document'.
 * Bundle: 'oe_document'.
```

Another option is to keep the comment simple and not have it include all the info, and instead we expect the reader to check the template name.
Tbh, I would not trust the comment anyway.

But, here, we actually can squeeze it into one line!
```suggestion
 * Template for the 'Document' field in the 'Document' document reference type.
```

Reply:
ok